modules/ask_va_api/spec/app/lib/ask_va_api/translator_spec.rb

```diff
@@ -36,8 +33,8 @@
 
     it 'translates all the option keys from name to id' do
       expect(result[:InquiryAbout]).to eq(translated_payload[:InquiryAbout])
-      expect(result[:InquirySource]).to eq(translated_payload[:InquirySource])
-      expect(result[:InquiryType]).to eq(translated_payload[:InquiryType])
+      # expect(result[:InquirySource]).to eq(translated_payload[:InquirySource])
+      # expect(result[:InquiryType]).to eq(translated_payload[:InquiryType])
```

Comment:
Can you remove these comments if they aren't needed

Reply:
Done